src/base/emu-i386/simx86/codegen-x86.c

```diff
@@ -3176,7 +3178,20 @@ unsigned int Exec_x86(TNode *G, int ln)
 "			rdtsc\n"
 			: "=a"(TimeStartExec.t.tl),"=d"(TimeStartExec.t.th)
 		);
-	__asm__ __volatile__ (
+
+	// fast inner loop: if the following conditions are not true
+	// we only need to call FindTree and NodeLinker, and check for
+	// signals, then go straight back to calling generated code
+#if defined(PROFILE) || defined(SINGLESTEP)
+	slow = 1;
+#else
+	slow = close || debug_level('e') || eTimeCorrect >= 0 ||
```

Comment:
`close` is a bad name for a variable, as
its a function name also. I think this may
lead to surprises, eg if someone removes
this parameter, that line won't error and
may be forgotten.
Also would be nice to add brackets around
this large logical expression.